py/tests/test_processing.py

```diff
@@ -128,6 +128,24 @@ def test_normalize_user_agent(must_normalize):
         assert "contexts" not in event
 
 
+def test_validate_pii_selector_spec():
+    sentry_relay.validate_pii_selector_spec("test")
+    sentry_relay.validate_pii_selector_spec("$user.id")
+    sentry_relay.validate_pii_selector_spec("extra.'sys.argv'.**")
+
+    with pytest.raises(ValueError) as e:
+        sentry_relay.validate_pii_selector_spec("no_spaces allowed")
+    assert "1 | no_spaces allowed" in str(e.value)
```

Comment:
Q: any reason to not assert the whole error message?

Reply:
I changed the error message to something more concise now and I am also asserting the full message now